etcdserver/config.go

```diff
@@ -58,10 +58,6 @@ func (c *ServerConfig) VerifyBootstrapConfig() error {
 		return fmt.Errorf("cannot use %x as member id", raft.None)
 	}
 
-	if c.DiscoveryURL == "" && !c.NewCluster {
-		return fmt.Errorf("initial cluster state unset and no wal or discovery URL found")
-	}
-
```

Comment:
The description of the error message is not accurate. And i think what it wants to check has been covered by `// Advertised peer URLs must match those in the cluster peer list` ten lines below.